nodes/nomos-node/src/config.rs

```diff
@@ -126,6 +121,15 @@ pub struct CryptarchiaArgs {
     note_nonce: Option<String>,
 }
 
+#[derive(Parser, Debug, Clone)]
+pub struct TimeArgs {
+    #[clap(long = "consensus-chain-start", env = "CONSENSUS_CHAIN_START")]
+    chain_start_time: Option<i64>,
```

Comment:
Why would this not be a `u64` instad of a `i64`?

Reply:
If there's some baked in assumption, it would be good to describe it in a comment.

Reply:
Because unix timestamps format are `i64` for the used `OffsetDateTime`.

Reply:
I think it would still make sense to include a comment explaining what a negative value means. Or is that specified elsewhere?

Reply:
Well, as its built from a timestamp in UNIX I actually assumed a negative value would mean datetimes previous to that. iirc it works like that (?)